static/app/views/replays/detail/userReplays/index.tsx

```diff
@@ -0,0 +1,207 @@
+import {useRef} from 'react';
+import {
+  AutoSizer,
+  CellMeasurer,
+  GridCellProps,
+  InfiniteLoader,
+  MultiGrid,
+} from 'react-virtualized';
+import styled from '@emotion/styled';
+
+import Placeholder from 'sentry/components/placeholder';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useRoutes} from 'sentry/utils/useRoutes';
+import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
+import NoRowRenderer from 'sentry/views/replays/detail/noRowRenderer';
+import useInfiniteLoader, {
+  LoadingStatus,
+} from 'sentry/views/replays/detail/useInfiniteLoader';
+import useLoadReplaysFromUser from 'sentry/views/replays/detail/userReplays/useLoadReplaysFromUser';
+import useVirtualizedGrid from 'sentry/views/replays/detail/useVirtualizedGrid';
+import HeaderCell from 'sentry/views/replays/replayTable/headerCell';
+import TableCell from 'sentry/views/replays/replayTable/tableCell';
+import {ReplayColumns} from 'sentry/views/replays/replayTable/types';
+import type {ReplayListRecord, ReplayRecord} from 'sentry/views/replays/types';
+
+type Props = {
+  replayId: undefined | string;
+  user: undefined | ReplayRecord['user'];
+};
+
+const COLUMNS = [
+  ReplayColumns.user,
+  ReplayColumns.startedAt,
+  ReplayColumns.duration,
+  ReplayColumns.countErrors,
+  ReplayColumns.activity,
+];
+
+function ReplaysFromUser({replayId, user}: Props) {
+  const routes = useRoutes();
+  const organization = useOrganization();
+
+  const {eventView, loadRows} = useLoadReplaysFromUser({user});
+
+  const {isLoading, isRowLoaded, loadMoreRows, rows, rowState} =
+    useInfiniteLoader<ReplayListRecord>({
```

Comment:
Note we don't use inf loading anywhere else in the application

Reply:
We kinda have the same ui pattern of scrolling down a long list though with the Replay network table, and console. Just that the data is pre-fetched. 

I had pagination buttons in here for a quick second, but it looked weird, the page is designed so the section extends flush to the bottom. we could design it another way, no rush to get this in.